tools/clang/lib/SPIRV/SpirvContext.cpp

```diff
@@ -361,6 +361,49 @@ SpirvDebugInstruction* SpirvContext::getDebugTypeFunction(
   return debugType;
 }
 
+SpirvDebugInstruction *
+SpirvContext::getDebugTypeTemplate(const SpirvType *spirvType,
+                                   SpirvDebugInstruction *target) {
+  // NOTE: Do not search it in debugTypes because we assume that
+  // only resource type e.g., RWStructuredBuffer<S> can be a DebugTypeTemplate
+  // and its DebugTypeComposite keeps this DebugTypeTemplate as a member.
+  auto it = debugTypes.find(spirvType);
```

Comment:
The comment says: Do _not_ search in debugTypes.
This is searching in debugTypes.